src/python/grpcio/grpc/compatibility.py

```diff
@@ -0,0 +1,6 @@
+import six
+
+if six.PY2:
```

Comment:
That should work, I just didn't realize `six.moves.cStringIO` was there. However, six.moves.cStringIO is a constructor (py2) / class (py3), not a module, is that alright?